pkg/utils/utils.go

```diff
@@ -183,11 +183,7 @@ func instantiateInformers(kubeClient KubeClientIntf, registeredInformers []strin
 		case EndpointInformer:
 			informers.EpInformer = kubeInformerFactory.Core().V1().Endpoints()
 		case SecretInformer:
-			if akoNSBoundInformer {
```

Comment:
akoNSBoundInformer,akoInformerFactory and akoNS can be removed from this function as they are not used.

Reply:
@swathinsankaran I have re-introduced this block again as in `advanceL4` deployment all this will be used. So i don't want to touch that functionality. So just removed `route` related stuff.